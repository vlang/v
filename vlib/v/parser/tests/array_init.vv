fn main() {
	_ := []int
	_ := [1]int
	_ := []!int{}
}
