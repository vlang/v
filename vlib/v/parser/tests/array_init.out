vlib/v/parser/tests/array_init.vv:2:7: warning: use `x := []Type{}` instead of `x := []Type`
    1 | fn main() {
    2 |     _ := []int
      |          ~~~~~
    3 |     _ := [1]int
    4 |     _ := []!int{}
vlib/v/parser/tests/array_init.vv:3:7: warning: use e.g. `x := [1]Type{}` instead of `x := [1]Type`
    1 | fn main() {
    2 |     _ := []int
    3 |     _ := [1]int
      |          ~~~~~~
    4 |     _ := []!int{}
    5 | }
vlib/v/parser/tests/array_init.vv:4:9: error: arrays do not support storing Result values
    2 |     _ := []int
    3 |     _ := [1]int
    4 |     _ := []!int{}
      |            ^
    5 | }